show-land/sass/main.scss

```diff
@@ -0,0 +1,5 @@
+@import"variables.scss";
+@import"base.scss";
+@import"button";
+@import"nav";
+@import"typography";
```

Comment:
First, work more on the beauty of the code, it will be much more beautiful
For example, you can use these:
```
=========================================================================================
*********
*******
*****
***
```